Weather App/ViewModel/Utility.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Utility.swift
+//  Weather App
+//
+//  Created by Shikhar Sharma on 22/03/22.
+//
+
+import Foundation
+
+extension String {
+    func getImageLocationFromUrl () -> String{
+        let ch = self[self.index(self.endIndex, offsetBy: -11)]
+        if ch == "d" {
+            return String(self[self.index(self.endIndex, offsetBy: -11)..<self.index(self.endIndex, offsetBy: -4)])
+        }
+        return String(self[self.index(self.endIndex, offsetBy: -13)..<self.index(self.endIndex, offsetBy: -4)])
```

Comment:
This will only work for valid cases and crash when these conditions are not met. You can introduce error handling here

Reply:
added guard, should take carer of invalid cases, and UIImage handles itself if valid named image is not found